tests/Fastapi/test_fastapi.py

```diff
@@ -0,0 +1,343 @@
+"""
+Copyright (c) 2020, VRAI Labs and/or its affiliates. All rights reserved.
+
+This software is licensed under the Apache License, Version 2.0 (the
+"License") as published by the Apache Software Foundation.
+
+You may not use this file except in compliance with the License. You may
+obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+License for the specific language governing permissions and limitations
+under the License.
+"""
+import asyncio
+import json
+
+from fastapi import FastAPI, Depends
+from fastapi.requests import Request
+from fastapi.responses import JSONResponse
+from fastapi.testclient import TestClient
+from pytest import fixture
+from pytest import mark
+
+from supertokens_python import init, session
+from supertokens_python.framework.fastapi.fastapi_middleware import Middleware
```

Comment:
ideally ,it should be something like:
```py
from supertokens_python.framework.fastapi import Middleware
```

You'll just need to update the __init__.py of fastapi framework for that

Reply:
This also applies to other frameworks as well